diagnostic.md

```diff
@@ -36,13 +48,25 @@ Now, we want to get a single donut from the server. Write the `curl` request
 you'd use to retrieve a single donut, using whatever ID you'd like.
 
 ```sh
-# your answer here
+curl --include --request "GET" "http://www.example.com/donuts$ID" \
```

Comment:
Almost. The actual curl request would be:

`curl --include --request "GET" "http://www.example.com/donuts/$ID"` You were missing the '/' before `$ID`